### PR TITLE
add script to pull data from datadog

### PR DESCRIPTION
**Pull request by sravfeyn**
@mkangia 
(Not sure where the best place for this to go is)

### CODE REVIEW

**Comment by mkangia** [comment]
@sravfeyn Can you share whats the use case?

Probably if icds specific can go under `custom/icds`?

**Comment by sravfeyn** [comment]
This is not really ICDS specific, it's a script to pull raw data from any datadog graph. Currently it's used to pull raw data for some ICDS speicifc datadog graphs

File: corehq/util/datadog/raw_data.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    )
+
+
+def print_csv_series(results):
+    # parse and print datadog query results in a CSV format
+    data = results['series'][0].get('pointlist', [])
+    print ", ".join(["date"] + METRICS.keys())
+    for i in range(0, len(data)):
+        posix_time = data[i][0]
+        datespan = datetime.fromtimestamp(posix_time / 1000)  # python datetime POSIX TZ issue
```
Comment (mkangia) [inline corehq/util/datadog/raw_data.py]: Can you add some detail about `python datetime POSIX TZ issue` for context.

**Review by mkangia — APPROVED** [review]
Looks good.

**Comment by snopoke** [comment]
Moved here: https://github.com/dimagi/metrics_analysis/commit/b8b6915a8fa8930c6cf3dc6f895a79471b74f9d5

**Pull request closed without merging**